README.md

```diff
@@ -38,6 +46,14 @@ docker run -d --rm --network=host \
   ghcr.io/jokob-sk/netalertx:latest
 ```
 
+To deploy a containerized instance directly from the source repository, execute the following BASH sequence:
+```bash
+git clone https://github.com/jokob-sk/NetAlertX.git
+cd NetAlertX
+docker compose up --force-recreate --build
+# To customize: edit docker-compose.yaml and run that last command again
+```
+
```

Comment:
Let's add a note that this is without persistent storage - e.g. all your config and your db will be wiped on container rebuild if you don't mount persistent storage.